base/domain-sot.yml

```diff
@@ -0,0 +1,380 @@
+    type: unfeaturized
+    auto_fill: false
+  city_location_coords:
+    type: unfeaturized
+    auto_fill: false
+  medical_condition:
+    type: unfeaturized
+    auto_fill: false
+  medical_condition_obesity:
+    type: unfeaturized
+    auto_fill: false
+  medical_condition_diabetes:
+    type: unfeaturized
+    auto_fill: false
+  medical_condition_hypertension:
+    type: unfeaturized
+    auto_fill: false
+  medical_condition_cardio:
+    type: unfeaturized
+    auto_fill: false
+  symptoms_fever:
+    type: unfeaturized
+    auto_fill: false
+  symptoms_cough:
+    type: unfeaturized
+    auto_fill: false
+  symptoms_sore_throat:
+    type: unfeaturized
+    auto_fill: false
+  symptoms_difficulty_breathing:
+    type: unfeaturized
+    auto_fill: false
+  symptoms_taste_smell:
+    type: unfeaturized
+    auto_fill: false
+  exposure:
+    type: unfeaturized
+    auto_fill: false
+  tracing:
+    type: unfeaturized
+    auto_fill: false
+  requested_slot:
+    type: unfeaturized
+
+forms:
+  - healthcheck_form
+  - healthcheck_terms_form
+  - healthcheck_profile_form
+
+responses:
+  utter_welcome:
+      - text: |
+          *HealthCheck* ke sesebediswa sa hao sa beke le beke sa tlhahlobo ya kotsi. Ka kopo re thuse ka ho araba dipotso tse mmalwa ka wena le bophelo ba hao bo botle. O tla fumana dikeletso mabapi le hore na ke mehato efe eo o ka e nkang ho latela tataiso ya naha mme dintlha tseo o di kenyang di tla re thusa ho etsa meralo ya naha ya ho lwantsha COVID-19. Re ya leboha ka ho nka bohato le ho kenya letsoho maphelong a baahi bohle LE ho emisa ho ata ha vaerase 🦠
+
+  utter_welcome_back:
+      - text: |
+          Welcome back to *HealthCheck*, your weekly risk assessment tool. Thank you for coming forward and for contributing to the health of all citizens AND stopping the spread of the virus 🦠
+
+  utter_ask_terms:
+      - text: |
+          Araba ka le leng la mantswe a ngotsweng ka BOTENYA ho tswela pele
+
+          AMOHELA ✅ - _ho amohela melawana le dipehelo_
+          TSE DING TSE NGATA 📄 - ho bala melawana le dipehelo_
+
+          (kapa Araba MENU 📌 _ ho kgutlela ho menu wa pele_)
+
+  utter_more_terms:
+      - text: |
+          O netefatsa hore wa ikarabella bakeng sa tlhokomelo ya hao ya bophelo bo botle le kalafo. Tshebeletso ena e fana ka lesedi feela.
+
+          Tshebeletso ena ha e nke sebaka sa dikeletso/tlhahlobo/kalafo tsa ngaka. Fumana dikeletso tsa mofani wa ditshebeletso tsa bophelo bo botle mabapi le boemo ba hao ba kalafo/tlhokomelo, haholo-holo haeba o ba le matshwao a matla.
+
+          O netefatsa hore ha wa lokela ho se tsotelle/ ho dieha ho batla boeletsi ba ngaka mabapi le kalafo/tlhokomelo ka lebaka la tshebeletso ena. Kotsi ya ho itshetleha leseding lena e boikarabellong ba hao.
+
+          Araba ka le leng la mantswe a ngotsweng ka *BOTENYA* ho tswela pele
+          AMOHELA ✅ - _ho amohela melawana le dipehelo_
+          (kapa Araba MENU 📌 _ ho kgutlela ho menu wa pele_)
```

Comment:
Keywords need to be bolded. Also the space after `_` might result in that section not being italicised.